Makefile

```diff
@@ -500,7 +500,7 @@ test-ci-js: | clear-stalled
 # Related CI jobs: most CI tests, excluding node-test-commit-arm-fanned
 test-ci: LOGLEVEL := info
 test-ci: | clear-stalled build-addons build-js-native-api-tests build-node-api-tests doc-only
-	out/Release/cctest --gtest_output=tap:cctest.tap
+	out/Release/cctest --gtest_output=xml:cctest.xml
```

Comment:
Is this going to break the CI build jobs that are expecting tap files (which remains the case for older releases)?

Reply:
https://ci.nodejs.org/job/node-test-commit-linuxone/12467/nodes=rhel72-s390x/console
```console
17:35:50 tap2junit: error: argument --input/-i: can't open 'cctest.tap': [Errno 2] No such file or directory: 'cctest.tap'
```

Reply:
Ah I guess `gtest` dropped tap support?

https://ci.nodejs.org/job/node-test-binary-windows-2/124/COMPILED_BY=vs2017,RUNNER=win10,RUN_SUBSET=addons/console
```console
17:42:51 [WARNING] c:\workspace\node-compile-windows\test\cctest\gtest\gtest-all.cc(6495):: WARNING: unrecognized output format "tap" ignored.
...
17:43:57 c:\workspace\node-test-binary-windows-2>if "1" == "1" (if not exist cctest.tap (
17:43:57 echo "Missing cctest.tap file"  
17:43:57  set "EXIT_RETURN_VALUE=1" 
17:43:57 ) ) 
17:43:57 "Missing cctest.tap file"
17:43:57 
17:43:57 c:\workspace\node-test-binary-windows-2>exit /b 1 
17:43:58 Build step 'Execute Windows batch command' marked build as failure
17:43:58 Run condition [Always] enabling perform for step [[Execute Windows batch command, Execute shell]]
```

Reply:
We anyway convert to xUnit format, so just need to move it to where Jenkins is expecting it
```
mv cctest.xml out/Release/ || true
```

Reply:
For Windows the output format needs to be changed https://github.com/nodejs/node/blob/427fce711f864f206a52ee8883442d443a8c4d34/vcbuild.bat#L87

Reply:
Done.